files/scripts/arp_update

```diff
@@ -7,12 +7,13 @@
 # Send gratuitous ARP/NDP requests to VLAN member neighbors to refresh
 # the ipv4/ipv6 neighbors state.
 
+ARP_UPDATE_VARS_FILE="/usr/share/sonic/templates/arp_update_vars.j2"
+
 while /bin/true; do
   # find L3 interfaces which are UP, send ipv6 multicast pings
-  echo "{% for (name, prefix) in INTERFACE|pfx_filter %} {{name}} {% endfor %}" > /tmp/intf_tmp.j2
-  INTERFACE=`sonic-cfggen -d -t /tmp/intf_tmp.j2`
-  echo "{% for (name, prefix) in PORTCHANNEL_INTERFACE|pfx_filter %} {{name}} {% endfor %}" > /tmp/pc_intf_tmp.j2
-  PC_INTERFACE=`sonic-cfggen -d -t /tmp/pc_intf_tmp.j2`
+  ARP_UPDATE_VARS=$(sonic-cfggen -d -t ${ARP_UPDATE_VARS_FILE})
```

Comment:
`INTERFACE `is from  INTERFACE table and not PORT and `PC_INTERFACE `is from PORTCHANNEL_INTERFACE and not PORTCHANNEL. Seems there is a mismatch in the `arp_update_vars.j2`

Reply:
OK, my config_db on the dut did not have INTERFACE table. Are we looking here for interface that are not portchannels? If not, can you please let me know what the INTERFACE contents are? Also, for PORTCHANNEL_INTERFACE, it has repeated contents that appeared unnecessary to me.

Reply:
The contents of INTERFACE would be same as PORTCHANNEL_INTERFACE. Both have IP addresses in key. Then "unnecessary" contents are filtered by this - `PORTCHANNEL_INTERFACE|pfx_filter`

Reply:
I don't see INTERFACE entry in the config_db.json. I tried a couple of devices. Where can I find it. The original script had interface generates empty list. For PORTCHANNEL_INTERFACE, there were duplicate entries in the pc_interface list. Before the ping6cmd is run, the interface is checked for `state UP`. There are differences, however is the new way of extracting info leading to the correct interfaces being extracted?

Reply:
No, the new way of extracting doesn't give the correct interfaces